ChangeLog.md

```diff
@@ -49,6 +49,8 @@ Other enhancements:
   i.e. `stack build --keep-tmp-files --ghc-options=-keep-tmp-files`.
   See [#3857](https://github.com/commercialhaskell/stack/issues/3857)
 * Improved error messages for snapshot parse exceptions
+* `stack unpack` now supports a `--to /target/directory` option to
```

Comment:
I think it would be worth clarifying that the `package-version` subdirectory will still be created. It's slightly ambiguous to me right now what the expected behavior is.